<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TinyMainWindow</class>
 <widget class="QMainWindow" name="TinyMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkyMPC</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,2,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="ServersComboBox" name="comboBox_servers">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_clock">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_minimize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Sleep Timer</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/image/close.svgz</normaloff>:/image/close.svgz</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_close">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Sleep Timer</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/image/close.svgz</normaloff>:/image/close.svgz</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="1" column="2">
       <widget class="QLabel" name="label_artist">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Artist</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources.qrc">:/image/appicon.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_title">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Artist</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolButton_play">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="toolTip">
         <string>Play</string>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/image/play.svgz</normaloff>:/image/play.svgz</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_stop">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="toolTip">
         <string>Stop</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/image/stop.svgz</normaloff>:/image/stop.svgz</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_sleep_timer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Sleep Timer</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/image/timer.svgz</normaloff>:/image/timer.svgz</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolButton_prev">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/image/prev.svgz</normaloff>:/image/prev.svgz</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_next">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/image/next.svgz</normaloff>:/image/next.svgz</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="action_random">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/image/random.svgz</normaloff>:/image/random.svgz</iconset>
   </property>
   <property name="text">
    <string>Ra&amp;ndom</string>
   </property>
   <property name="toolTip">
    <string>Random</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_repeat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/image/repeat.svgz</normaloff>:/image/repeat.svgz</iconset>
   </property>
   <property name="text">
    <string>&amp;Repeat</string>
   </property>
   <property name="toolTip">
    <string>Repeat</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_debug">
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="action_play">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/image/play.svgz</normaloff>:/image/play.svgz</iconset>
   </property>
   <property name="text">
    <string>&amp;Play</string>
   </property>
   <property name="toolTip">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/image/stop.svgz</normaloff>:/image/stop.svgz</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="action_previous">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/image/prev.svgz</normaloff>:/image/prev.svgz</iconset>
   </property>
   <property name="text">
    <string>Pre&amp;vious</string>
   </property>
   <property name="toolTip">
    <string>Previous</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="action_next">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/image/next.svgz</normaloff>:/image/next.svgz</iconset>
   </property>
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="toolTip">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_help_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_edit_copy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_edit_cut">
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_edit_paste_bottom">
   <property name="text">
    <string>Paste (&amp;Bottom)</string>
   </property>
   <property name="toolTip">
    <string>Paste (Bottom)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_edit_delete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_network_connect">
   <property name="text">
    <string>&amp;Connect...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_network_disconnect">
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="action_single">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/image/single.svgz</normaloff>:/image/single.svgz</iconset>
   </property>
   <property name="text">
    <string>S&amp;ingle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_playlist_edit">
   <property name="text">
    <string>&amp;Edit...</string>
   </property>
   <property name="toolTip">
    <string>Edit</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="action_edit_paste_insert">
   <property name="text">
    <string>Paste (&amp;Insert)</string>
   </property>
   <property name="toolTip">
    <string>Paste (Insert)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_playlist_add_location">
   <property name="text">
    <string>&amp;Add location...</string>
   </property>
   <property name="toolTip">
    <string>Add location</string>
   </property>
  </action>
  <action name="action_playlist_update">
   <property name="text">
    <string>&amp;Update</string>
   </property>
   <property name="toolTip">
    <string>Update</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_file_close">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_playlist_quick_save_1">
   <property name="text">
    <string>Quick save 1</string>
   </property>
   <property name="shortcut">
    <string>Shift+F7</string>
   </property>
  </action>
  <action name="action_playlist_quick_save_2">
   <property name="text">
    <string>Quick save 2</string>
   </property>
   <property name="shortcut">
    <string>Shift+F8</string>
   </property>
  </action>
  <action name="action_playlist_quick_load_1">
   <property name="text">
    <string>Quick load 1</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="action_playlist_quick_load_2">
   <property name="text">
    <string>Quick load 2</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="action_playlist_clear">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
   <property name="shortcut">
    <string>Shift+F6</string>
   </property>
  </action>
  <action name="action_playlist_unify">
   <property name="text">
    <string>Unify</string>
   </property>
  </action>
  <action name="action_consume">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/image/consume.svgz</normaloff>:/image/consume.svgz</iconset>
   </property>
   <property name="text">
    <string>Consume</string>
   </property>
  </action>
  <action name="action_network_reconnect">
   <property name="text">
    <string>Reconnect</string>
   </property>
   <property name="shortcut">
    <string>Shift+F5</string>
   </property>
  </action>
  <action name="action_sleep_timer">
   <property name="text">
    <string>Sleep &amp;timer...</string>
   </property>
   <property name="shortcut">
    <string>Shift+F1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ServersComboBox</class>
   <extends>QComboBox</extends>
   <header>ServersComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
